working/wildcards/feature-specification.md

```diff
@@ -85,6 +89,10 @@ A *local declaration* is any of:
     Foo(_, this._, super._, void _(), {_}) {}
 
     list.where((_) => true);
+
+    void f(void g(_, _)) {}
+
+    typedef T = void Function(String _, String _);
```

Comment:
Ah, good catch. I didn't think about parameters in function types since you can't use them. But the language does still report a collision on them if you have a duplicate and this proposal should remove that error.